src/troute-config/troute/config/compute_parameters.py

```diff
@@ -82,7 +82,6 @@ class HybridParameters(BaseModel, extra='forbid'):
     run_hybrid_routing: bool
     # NOTE: required for hybrid simulations
     diffusive_domain: Optional[FilePath] = None
-
```

Comment:
This looks like just a white space change? Probably not needed.

Reply:
Sure.